syntax = "proto2";

option java_package = "com.nextev.cvs_proto.protobuf.events";
option java_outer_classname = "EcallErrorServiceMessage";

//sub type:"ecall_error_event"
message EcallErrorEvent {
    optional string id = 1; // unique id, VIN for vehicle data
    optional int32 version = 2; // message protocol version
    optional uint64 sample_ts = 3; // status change time
    optional int32 emgc_call_flr_sts = 4; //the real value CGW received from Ecall box
                                          //3 "Automatic eCall disabled" 2 "eCall primary fault" 1 "eCall minor fault" 0 "eCall no fault" ;

}